cmake/BLTOptions.cmake

```diff
@@ -53,8 +53,14 @@ option(ENABLE_COVERAGE   "Enables code coverage support" OFF)
 ################################
 # Build Options
 ################################
+get_property(_languages GLOBAL PROPERTY ENABLED_LANGUAGES)
+if(_languages MATCHES "Fortran")
```

Comment:
I think ``ENABLED_LANGUAGES`` is a list -- https://cmake.org/cmake/help/v3.8/prop_gbl/ENABLED_LANGUAGES.html#prop_gbl:ENABLED_LANGUAGES

Does this only work if Fortran is the only enabled language?  
What if multiple languages are enabled?

Reply:
Since one of our core macros already uses ``if( ... IN_LIST ...)``  (see the ``blt_setup_target`` macro), I think it is safe to say that blt already requires cmake 3.3 or higher.  

We should probably note this in our documentation, add a check in ``SetupBLT`` and set policy ``CMP0057`` appropriately. 

See: https://stackoverflow.com/questions/23323147/best-way-to-check-with-cmake-whether-list-containts-a-specific-entry  for some other options for checking if an item is in a list.

Reply:
Yes, `ENABLED_LANGUAGES` IS a list.

Fortunately `MATCHES` takes a regex so this check just looks for `Fortran` any where in the "list" (i.e. string with semicolons in it)

I noticed you were using IN_LIST, but I didn't see any calls to deal with CMP0057. I think it would be wise/safe to set CMP0057 to NEW.

I agree, IN_LIST is a much better solution.

Reply:
To further elaborate, IN_LIST was causing CI failures because, AFAICT, CMP0057 was not set anywhere. My preference would be to set `CMP0057` to `NEW` and switch back to `IN_LIST` (revert eef788c49317982b0341512a9276adbf4925e194)

Reply:
I created issue #109 about the required version of cmake and the ``CMP0057`` issue.